query_postgis.py

```diff
@@ -9,10 +9,15 @@
 import psycopg2
 import pandas as pd
 
+from rtree import index
+from shapely.wkt import loads
 
-POSTGIS_USER = os.environ["PGUSER"]
-POSTGIS_PASSWORD = os.environ["PGPASSWORD"]
```

Comment:
Please remember to undo any changes, that you needed for your local environment. In the optimal case we should not need to do that.